config/config/wc-config.yaml

```diff
@@ -113,6 +113,7 @@ prometheus:
   ## Additional prometheus scrape config.
   ## ref: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config
   additionalScrapeConfigs: []
+  predictLinearLimit: 66
```

Comment:
Could you add a comment above explaining this config parameter? I'm sure I'll forget next week. :smile: